x/dao/keeper/reissuance_test.go

```diff
@@ -4,20 +4,21 @@ import (
 	"fmt"
 	"testing"
 
+	"github.com/planetmint/planetmint-go/config"
 	keepertest "github.com/planetmint/planetmint-go/testutil/keeper"
 	"github.com/stretchr/testify/assert"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
-	"github.com/planetmint/planetmint-go/x/dao/keeper"
+	DaoKeeper "github.com/planetmint/planetmint-go/x/dao/keeper"
```

Comment:
interesting that this works, but please use all lower case `daokeeper`, because that is the syntax we use in the rest of the files